src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -47,24 +99,28 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
   };
 
   /**
-   * Updates metaform field VisiblityIf parameters
+   * Updates metaform field or section visibleIf parameters
    *
-   * @param newMetaformField new metaform field
-   * @param eventValue FieldCondition value
+   * @param newMetaformUpdateFieldOrSection metaform field or section
+   * @param value event condition equals value (kylla / ei)
    */
-  const updateFormFieldVisiblityValues = (newMetaformField: MetaformField, eventValue: string) => {
-    setRequiredConditionFieldBoolean(eventValue);
-    setRequiredConditionInfoField(false);
-    setDisabledValue(false);
-    setRequiredConditionField("");
-    setRequiredConditionFieldBoolean("");
-    if (sectionIndex === undefined || fieldIndex === undefined) {
+  const updateFormFieldOrSectionVisiblityValues = (newMetaformUpdateFieldOrSection: any, value: string) => {
```

Comment:
Why any type?

Reply:
I had dublicate problem in first commit where i had 2 select menus and it showed field or section select menu depending situation. 

But if that is like
newMetaformUpdateFieldOrSection: metaformField | metaformSection.
its gonna give error because Property 'type' is missing in type 'MetaformSection'